user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectSummaryComponent.java

```diff
@@ -560,6 +570,9 @@ private void handleSuccess(ProjectUpdateResponse response) {
   private void handleRequestFailed(RequestFailedException error) {
```

Comment:
We probably have to remove the request or timestamp it once it resulted in a successful response.